packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -238,23 +221,48 @@ export const ChatList = ({
   ].includes(chatListState?.status ?? '');
 
   const isError = ['server connection lost', 'error'].includes(chatListState?.status ?? '');
+  const targetElementRef = useRef(null);
+
+  useEffect(() => {
+    const handleIntersection = (entries: IntersectionObserverEntry[]) => {
+      entries.forEach((entry: IntersectionObserverEntry) => {
+        // Check if the element is intersecting
+        if (entry.isIntersecting) {
+          // The element has come into view, you can perform your actions here
+          if (chatListState?.moreChatThreadsToLoad) {
+            chatListClient?.loadMoreChatThreads();
+          }
+        }
+      });
+    };
+    // Create a new Intersection Observer instance
+    const observer = new IntersectionObserver(handleIntersection, {
+      root: null, // observing intersections with the viewport
+      rootMargin: '0px',
+      threshold: 0.1 // Callback is invoked when 10% of the target is visible
```

Comment:
Why not go down to 1% or something? Because this is less than 10% and it kind of seems like it should load.

<img width="760" alt="image" src="https://github.com/plasne/microsoft-graph-toolkit/assets/15094473/72db8033-9a41-4a69-8b79-92a8a38e8698">